templates/todo/projects/csharp-cosmos-sql/.repo/bicep/infra/main.bicep

```diff
@@ -24,6 +24,9 @@ param logAnalyticsName string = ''
 param resourceGroupName string = ''
 param webServiceName string = ''
 
+@description('Flag to use Azure API Management to mediate the calls between the frontend and the backend API')
+param useAPIM bool = true
```

Comment:
This should be false.

Reply:
Yes, I just put it as true for testing. The wizard should ask for this parameter?

Reply:
could we set the APIM switch on the main.parameters.json file?

Reply:
It's set to false now